### PR TITLE
Add number sign in front of profile rankings

### PR DESCRIPTION
**Pull request by Joehuu**
---

### CODE REVIEW

**Comment by peppy** [comment]
Please check CI issues.

File: osu.Game/Overlays/Profile/Header/DetailHeaderContainer.cs [hunk 1]
```diff
@@ -148,8 +148,8 @@ private void updateDisplay(User user)
             foreach (var scoreRankInfo in scoreRankInfos)
                 scoreRankInfo.Value.RankCount = user?.Statistics?.GradesCount[scoreRankInfo.Key] ?? 0;
 
-            detailGlobalRank.Content = user?.Statistics?.Ranks.Global?.ToString("#,##0") ?? "-";
```
Comment (jorolf) [inline osu.Game/Overlays/Profile/Header/DetailHeaderContainer.cs]: You can include the sign in the original ``ToString()`` call

File: osu.Game/Overlays/Profile/Header/DetailHeaderContainer.cs [hunk 2]
```diff
@@ -148,8 +148,8 @@ private void updateDisplay(User user)
...
-            detailCountryRank.Content = user?.Statistics?.Ranks.Country?.ToString("#,##0") ?? "-";
+            detailGlobalRank.Content = user?.Statistics?.Ranks.Global == null ? "-" : $"#{user?.Statistics?.Ranks.Global?.ToString("#,##0")}";
```
Comment (jorolf) [inline osu.Game/Overlays/Profile/Header/DetailHeaderContainer.cs]:
```suggestion
            detailGlobalRank.Content = user?.Statistics?.Ranks.Global?.ToString("\\##,##0") ?? "-";
```

File: osu.Game/Overlays/Profile/Header/DetailHeaderContainer.cs [hunk 3]
```diff
@@ -148,8 +148,8 @@ private void updateDisplay(User user)
...
+            detailCountryRank.Content = user?.Statistics?.Ranks.Country == null ? "-" : $"#{user?.Statistics?.Ranks.Country?.ToString("#,##0")}";
```
Comment (jorolf) [inline osu.Game/Overlays/Profile/Header/DetailHeaderContainer.cs]:
```suggestion
            detailCountryRank.Content = user?.Statistics?.Ranks.Country?.ToString("\\##,##0") ?? "-";
```

**Review by jorolf — CHANGES_REQUESTED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**